examples/jsm/loaders/USDZLoader.js

```diff
@@ -503,6 +549,7 @@ class USDZLoader extends Loader {
 					const sampler = data[ 'def Shader "normal_texture"' ];
 
 					material.normalMap = buildTexture( sampler );
+					material.normalMap.colorSpace = LinearSRGBColorSpace;
```

Comment:
Nit: These non-color textures should use `NoColorSpace`. The result is the same now, but `LinearSRGBColorSpace` will cause an unwanted conversion under a different working color space. See https://github.com/mrdoob/three.js/pull/26644.

Reply:
Fixed – thank you! It's great that you started the big task of supporting wide gamut in three 🚀